content/content-tool/tool/src/java/org/sakaiproject/content/tool/FilePickerAction.java

```diff
@@ -881,9 +901,31 @@ else if(contentService.isDropboxMaintainer())
 			// user wants copies in hidden attachments area
 			template = TEMPLATE_ATTACH;
 		}
+		
+		// ONEDRIVE
+		if(onedriveOn) {
```

Comment:
does it make sense to say if cloud storage on (in the future there will be other cloud storage providers)?

Reply:
I agree but now we only have one implementation, we can refactor this part after adding a second implementation.